src/credentials/CHIPCert.cpp

```diff
@@ -935,5 +935,47 @@ CHIP_ERROR ConvertECDSASignatureRawToDER(P256ECDSASignatureSpan rawSig, ASN1Writ
     return err;
 }
 
+CHIP_ERROR ExtractPeerIdFromOpCert(const ChipCertificateData & opcert, PeerId * peerId)
+{
+    // Since we assume the cert is pre-validated, we are going to assume that
```

Comment:
Suggest pre-setting NodeID and FabricID to zero in peerId in case the prevalidated assumption fails.

Reply:
PeerId constructor should already pre-initialize.

Reply:
If we think this can in fact be an issue, we could just track whether we found both ids and return failure if we didn't....  I'll just go ahead and do that.

Reply:
> PeerId constructor should already pre-initialize.

It does, yes.